extensions/typescript-language-features/src/languageFeatures/hover.ts

```diff
@@ -33,7 +33,7 @@ class TypeScriptHoverProvider implements vscode.HoverProvider {
 			return undefined;
 		}
 
-		const enableExpandableHover = vscode.workspace.getConfiguration('typescript').get('experimental.expandableHover');
+		const enableExpandableHover = vscode.workspace.getConfiguration('typescript').get<boolean>('experimental.expandableHover', true);
```

Comment:
Please also update the `package.json`. Otherwise looks good

Reply:
Done.